front-end/src/Navbar.jsx

```diff
@@ -9,35 +10,78 @@ import {
     MenuButton,
     MenuList,
     MenuItem,
-    IconButton
+    IconButton,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalCloseButton,
+    ModalBody,
+    ModalFooter,
+    useDisclosure,
+    Slider,
+    SliderMark,
+    SliderTrack,
+    SliderFilledTrack,
+    Tooltip,
+    SliderThumb
 } from "@chakra-ui/react";
-
 import { HamburgerIcon, LockIcon, SunIcon, AddIcon } from '@chakra-ui/icons';
 
 function Navbar({ username, page }) {
     const navigate = useNavigate();
+    const { isOpen, onOpen, onClose: closeModal } = useDisclosure();
+    const [moodValue, setMoodValue] = useState(50);
+    const [danceValue, setDanceValue] = useState(50);
+    const [mixValue, setMixValue] = useState(50);
+    const [exploreValue, setExploreValue] = useState(50);
+    const [showMoodValue, setShowMood] = useState(false);
+    const [showDanceValue, setShowDance] = useState(false);
+    const [showMixValue, setShowMix] = useState(false);
+    const [showExploreValue, setShowExplore] = useState(false);
+
+    const labelStyles = {
+        mt: '2',
+        ml: '-2.5',
+        fontSize: 'sm',
+        color: 'black'
+    }
+
 
     const handleHome = () => {
         navigate(`/${username}`);
     };
 
-    const handleCreate = async () => {
-        if (page === "create") {
-            return;
-        }
+    const handleModalClose = () => {
+        setMoodValue(50);
+        setDanceValue(50);
+        setMixValue(50);
+        setExploreValue(50);
+        closeModal();
+    };
 
-        const stitchId = await createStitch();
+    const handleSubmit = async () => {
+        console.log('Form submitted with values:', { moodValue, danceValue, mixValue, exploreValue });
```

Comment:
Please do not submit `console.log` variables.

Reply:
Noticed that after committing, now it's gone thank you!